common/changes/@itwin/viewer-react/leck-ecsql-widget_2022-08-15-20-49.json

```diff
@@ -0,0 +1,10 @@
+{
+  "changes": [
+    {
+      "packageName": "@itwin/viewer-react",
```

Comment:
we dont want this in the viewer modules itself, just in the test app

Reply:
Do you mean move the provider to inside the test app instead of in viewer-react?

Reply:
@achrysaetos Idk where youre comment went, i dont see it in the pr
![image](https://user-images.githubusercontent.com/11051042/184966797-bcb73c39-7348-4bbd-9f5b-b1ce592a5119.png)

But yes. This should not be apart of the viewer-react pkg. Just needs to be apart of the test app. So can just move the code from the module into the app